manifests/init.pp

```diff
@@ -30,9 +30,9 @@
 #
 # $email_smtp_password::          Password for SMTP server auth, if authentication is enabled
 #
-# $locations_enabled::            Enable locations?
+# $locations_enabled::            (Deprecated) Enable locations?
```

Comment:
We typically don't deprecate in this way. I'd prefer to move it to `Advanced parameters` and write that it's useless starting 1.21.

Reply:
updated and moved